stock_scanner_web/views/web_templates.xml

```diff
@@ -0,0 +1,160 @@
+
+                    <t t-call-assets="stock_scanner_web.assets_frontend" t-css="false"/>
+                </body>
+            </html>
+        </template>
+
+        <template id="scanner_call" name="header" page="True">
+            <t t-call="stock_scanner_web.layout">
+                <div id="wrap" class="oe_structure oe_empty">
+                    <div class="row">
+                        <div class="col-xs-12 text-center">
+                            <h3>
+                                <span t-esc="code"/>
+                            </h3>
+                            <p t-if="code == 'L'">
+                                <table class="table table-condensed">
+                                    <thead>
+                                        <tr>
+                                            <th>Index</th>
+                                            <th>Menu</th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        <tr t-foreach="result" t-as="line">
+                                            <td>
+                                                <span t-esc="result.index(line)"/>
+                                            </td>
+                                            <td t-if="len(line) != 2 and line[0] != '|'">
+                                                <a id="btn-go" t-attf-href="/scanner_call/{{terminal_number}}/action/{{line}}" class="btn">
+                                                    <span t-esc="line"/>
+                                                </a>
+                                            </td>
+                                            <td t-if="len(line) != 2 and line[0] == '|'">
+                                                <span t-esc="line[1:]"/>
+                                            </td>
+                                            <td t-if="len(line) == 2 and line[0] != '|'">
+                                                <a id="btn-go" t-attf-href="/scanner_call/{{terminal_number}}/action/{{line[0]}}" class="btn">
+                                                    <span t-esc="line[1]"/>
+                                                </a>
+                                            </td>
+                                        </tr>
+                                    </tbody>
+                                </table>
+                            </p>
+                            <p t-if="code in ['M','E']">
+
+                                <h3>
+                                    <span t-esc="result[0][1:]"/>
+                                </h3>
+                                <t t-foreach="result[1:]" t-as="line">
+                                    <p t-esc="line"/>
+                                </t>
+                                <a id="btn-go" t-attf-href="/scanner_call/{{terminal_number}}/action/0" class="btn">
```

Comment:
You forgot to test the value : a default value set to True should call the 'back' action, instead of 'action'.